es-spatial-plugin/src/cmr/elasticsearch/plugins/spatial/script/core.clj

```diff
@@ -28,23 +30,46 @@
 ;;                         Begin script helper functions                     ;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(def parameters
+  "The parameters to the Spatial script"
+  [:ords :ords-info :operator])
+
+(defn- extract-params
+  "Extracts the parameters from the params map given in the script."
+  [script-params]
+  (when script-params
+    (into {} (for [param parameters]
+               [param (XContentMapValues/nodeStringValue
+                        (get script-params (name param)) nil)]))))
+
 (defn- get-from-fields
   [^LeafStoredFieldsLookup lookup key]
   (when (and lookup key (.containsKey lookup key))
     (when-let [^FieldLookup field-lookup (.get lookup key)]
       (seq (.getValues field-lookup)))))
 
+(defn remove-br
+  "Removes bounding rectangles from granule spatial data."
+  [shapes]
+  (let [shapes-no-br (into [] (remove #(instance? cmr.spatial.mbr.Mbr %) shapes))]
+    shapes-no-br))
+
 (defn doc-intersects?
   "Returns true if the doc contains a ring that intersects the ring passed in."
-  [^LeafStoredFieldsLookup lookup intersects-fn]
+  [^LeafStoredFieldsLookup lookup params intersects-fn]
   ;; Must explicitly return true or false or elastic search will complain
   (if-let [ords-info (get-from-fields lookup "ords-info")]
     (let [ords (get-from-fields lookup "ords")
-          shapes (srl/ords-info->shapes ords-info ords)]
+          shapes (srl/ords-info->shapes ords-info ords)
+          shapes-no-br (remove-br shapes)
```

Comment:
Change this line to `shapes-no-br (remove #(instance? cmr.spatial.mbr.Mbr %) shapes)`, and remove the function `remove-br`.

Reply:
Fixed